docs/DEVELOPER.md

```diff
@@ -1,12 +1,42 @@
 # Developer Documentation
 
+## Contributing
+Please read the [contributing guidelines here](https://github.com/graycore/daffodil/blob/master/CONTRIBUTING.md).
+
+## Build
+
+Run `ng build core` to build the @daffodil/core project. The build artifacts will be stored in the `dist/` directory. Use the `-prod` flag for a production build.
+
+## Release
+
+After updating the version and navigating to the dist folder, run `npm publish` to publish the current version of @daffodil/core to npm.
+
+## Running unit tests
+
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+To run an individual suite:
+
+```
+ng test core
+ng test foundation-demo
+```
+
 ## Running the example foundation-demo
-1. Ensure that lerna is installed in your local environment `npm install --global lerna`.
-2. Run `lerna bootstrap`. This will basically run `npm install` in all required packages within the project. 
-3. Compile the `@daffodil/core` project by running `ng build core`
-    * This will compile the `@daffodil/core` module into an npm package in the `dist` folder, identical to the one located in the npm `@daffodil/core` repository. 
+
+### Prerequisites
+* `Lerna` - `npm install --g lerna`.
+
```

Comment:
I think @angular/cli is another prerequisite that should be added - `npm install --g @angular/cli`

Reply:
@lderrickable added.